src/apic-extension/azext_apic_extension/aaz/latest/apic/service/_import_from_apim.py

```diff
@@ -17,8 +17,14 @@
 class ImportFromApim(AAZCommand):
     """Imports APIs from an Azure API Management service instance.
 
-    :example: Import From APIM
-        az apic service import-from-apim -g api-center-test --service-name contosoeuap --source-resource-ids '/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/servicegroup/providers/Microsoft.ApiManagement/service/contoso/apis/contosoapi'
+    :example: Import all APIs from APIM in same resource group
```

Comment:
@juliajuju93 can you help review the examples here? Please leave comments or approve the PR if they look good to you.